dex/msgjson/types.go

```diff
@@ -578,9 +581,10 @@ type TradeNote struct {
 
 // OrderNote is part of a notification about any type of order.
 type OrderNote struct {
-	Seq      uint64 `json:"seq,omitempty"`      // May be empty when part of an OrderBook.
-	MarketID string `json:"marketid,omitempty"` // May be empty when part of an OrderBook.
-	OrderID  Bytes  `json:"oid"`
+	Seq        uint64 `json:"seq,omitempty"`      // May be empty when part of an OrderBook.
+	MarketID   string `json:"marketid,omitempty"` // May be empty when part of an OrderBook.
+	OrderID    Bytes  `json:"oid"`
+	Commitment Bytes  `json:"com"`
```

Comment:
I'll update the spec in https://github.com/decred/dcrdex/pull/154

Reply:
Ah, I actually put the commitment in [`EpochOrderNote`](https://github.com/decred/dcrdex/pull/145/files#diff-9a36b4e081b2c1cc9032524815e5d57cR598) since the commitment only seems relevant there (not book/unbook).